data_dictionary/data_dictionary.cc

```diff
@@ -312,7 +328,10 @@ std::map<sstring, sstring> storage_options::to_map() const {
         [&ret] (const storage_options::s3& v) {
             ret.emplace("bucket", v.bucket);
             ret.emplace("endpoint", v.endpoint);
-        }
+        },
+        [&ret] (const storage_options::shared_filesystem& v) {
+            ret.emplace("dir", v.dir);
```

Comment:
Perhaps it's better to avoid abbreviation here, and use 'directory' instead.

Reply:
sure, I'll expand to "directory"